doc/classes/CubeMap.xml

```diff
@@ -33,6 +36,7 @@
 			<argument index="0" name="side" type="int" enum="CubeMap.Side">
 			</argument>
 			<description>
+				Return an Image for a side of the CubeMap using the CubeMap.Side enum or an integer 0-5.
```

Comment:
- Return -> Returns
- Link to other classes: [Image]

Returns an [Image] of the CubeMap's given side. See [code]SIDE_*[/code] constants.